.pubnub.yml

```diff
@@ -1,9 +1,33 @@
 ---
 name: swift
 scm: github.com/pubnub/swift
-version: "1.2.1"
+version: "2.0.0"
```

Comment:
Must be a pretty big change then.

Reply:
I wasn't sure if we're following semantic versioning or doing feature based versioning.  It's a semantic major version increase, but feature wise it doesn't add anything special.

Reply:
I'm fine with this, we should be following semantic versioning, it's just not "officially" implemented yet. Don't forget to create a gist or whatever for documentation changes.

Reply:
Sounds good; working on the docs update.